docs/readme.md

```diff
@@ -0,0 +1,9 @@
+# Taxonomy
+
+## Unclassified Tree
+
+Nodes are deemed *unclassified* when their *scientific name* contains (ignore case) the word `unclassified`; these generate the **Unclassified** tree.
+
+## Environmental Samples Tree
+
+The **Environmental Samples** tree is generated by the nodes whose *scientific name* has prefix (ignore case) `environmental samples`.
```

Comment:
Note that *scientific name* is the one and only name that api.ncbitaxonomy gives you.